src/api_manager/method_impl.cc

```diff
@@ -110,7 +110,12 @@ const std::string &MethodInfoImpl::first_authorization_url() const {
 
 void MethodInfoImpl::process_backend_rule(
     const ::google::api::BackendRule &rule) {
-  backend_address_ = rule.address();
+  // Strip the last "/", in case the address is mis-configured.
+  if (rule.address().back() == '/') {
```

Comment:
what happen with the last slash with a valid path, such as

https://foo.com/path/

Reply:
Updated, only strip it for APPEND_PATH_TO_ADDRESS case.